prow/scripts/cluster-integration/README.md

```diff
@@ -0,0 +1,17 @@
+# Cluster Integration
+
+## Overview
+
+The folder contains scripts that are responsible for performing a cleanup of Google Cloud Platform (GCP) objects built as part of the integration tests that run on pull requests.
```

Comment:
Scripts in that folder are not only responsible for cleanup. For example `pr-gke-integration.sh` create cluster.